<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Card Number Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>33</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Card Number Generator</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>172</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="howManyLabel">
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>How Many?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="howManyTextEdit">
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="bankNameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bank Name:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chooseBankName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="cardTypeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Card Type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chooseCardType">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>AMEX_CENTURION</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AMEX_PLATINUM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AMEX_GOLD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AMEX_OPTIMA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AMEX_CORPORATE_CARD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AMEX_BLUE_FOR_BUSINESS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AMEX_SMALL_CORPORATE_CARD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>N/A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CREDIT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DEBIT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PREPAID</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="displayTable">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>601</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
